apps/react-app/src/common-app/components/navbar/navbar.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { css } from '@emotion/css';
+import { theme } from '@/core/theme';
+
+export const nav = css`
+  position: fixed;
+  z-index: 999;
+  top: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: ${theme.palette.dark[900]};
+  height: 50px;
+`;
+
+export const logo = css`
+  margin-left: ${theme.spacing(4)};
+  height: ${theme.spacing(8)};
+
+  @media (min-width: 725px) {
+    background-image: url('/assets/logo-with-name.svg');
+    background-repeat: no-repeat;
```

Comment:
we only need to change image, background-repeat and background-size are the same (read next comment about media query)